bin/tape

```diff
@@ -25,6 +26,17 @@ opts.require.forEach(function(module) {
     }
 });
 
+if (opts.number !== false) {
+    var tape;
+    try {
+        tape = require(resolveModule("tape", { basedir: cwd }));
+    }
+    catch(err) {
+        tape = require("../"); // assume testing tape module itself
```

Comment:
By way of explanation, the `-n` option must inform the tape module of the number of the test to run. The above `resolveModule()` call requires that a module named `tape` have been installed. No module named `tape` has been installed when we run the test suite for `tape` itself. In order to be able to test the `-n` option within the `tape` module itself, it's necessary to load the module in the way that is required by the test suite. You'll find all the tests loading the `tape` module this way as well (not just mine). Without this line, the `tape` module would not be able to include tests of the `-n` option.